packages/at_commons/lib/src/verb/enroll_params.dart

```diff
@@ -1,4 +1,6 @@
 import 'package:json_annotation/json_annotation.dart';
+
+import '../../at_commons.dart';
```

Comment:
remove at_commons import and add specific import

Reply:
change made as part of [feddc9e](https://github.com/atsign-foundation/at_libraries/pull/530/commits/feddc9e80d5327223fa0a5fed5f6ff9a3173fe56)